### PR TITLE
resolve tasks in single mode

### PR DESCRIPTION
**Pull request by elahehrashedi**

### CODE REVIEW

File: Extension/src/LanguageServer/cppBuildTaskProvider.ts [hunk 1]
```diff
@@ -55,6 +55,12 @@ export class CppBuildTaskProvider implements TaskProvider {
         return undefined;
     }
 
+    public resolveInsiderTask(_task: CppBuildTask): CppBuildTask | undefined {
```
Comment (sean-mcmanus) [inline Extension/src/LanguageServer/cppBuildTaskProvider.ts:58]: Should the "resolveTask" method be removed? It doesn't appear to be used anymore.

Comment (elahehrashedi) [reply Extension/src/LanguageServer/cppBuildTaskProvider.ts:58]: It is used by VsCode when running tasks from tasks.json

Comment (sean-mcmanus) [reply Extension/src/LanguageServer/cppBuildTaskProvider.ts:58]: Ok

**Review by sean-mcmanus — APPROVED** [review]

File: Extension/src/LanguageServer/cppBuildTaskProvider.ts (hunk 1 above)
Comment (sean-mcmanus) [reply Extension/src/LanguageServer/cppBuildTaskProvider.ts:58]: TypeScript doesn't show any indication that the method is overriding a base class method (unlike C++'s override/virtual), so I didn't notice it was an override.

**Review by sean-mcmanus — APPROVED** [review]

**Pull request merged**